HW34/src/main/java/org/my/homework/Cart.java

```diff
@@ -0,0 +1,49 @@
+package org.my.homework;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope("prototype")
+public class Cart {
+
+    @Autowired
+    private List<Product> products;
```

Comment:
This is not a bean - no autowired and in general the logic that stands behind this List must be in ProductRepository